keps/sig-testing/3041-node-conformance-and-features/README.md

```diff
@@ -0,0 +1,782 @@
+and don’t actually need the `Feature` label any longer. Going forward, features
+that require a special environment outside of predefined `LinuxOnly`, `Serial`, etc.
+will need to define their "custom" labels and be commented with the description
+of the special environment needed.
+
+### Clean up labels that means whether special environment is needed
+
+#### NodeSpecialFeature
+
+The label `NodeSpecialFeature:` was introduced in [this document](https://docs.google.com/document/d/1BdNVUGtYO6NDx10x_fueRh_DLT-SVdlPC_SsXjYCHOE/edit?usp=sharing),
+but was never consistently used. Today we only have a couple uses of it that
+might be cleaned up in favor of using `Feature:` tag when applicable.
+
+#### NodeFeature
+
+`NodeFeatures` label was [introduced](https://docs.google.com/document/d/1BdNVUGtYO6NDx10x_fueRh_DLT-SVdlPC_SsXjYCHOE/edit?usp=sharing)
+to indicate that the feature may not work the same way on different container
+runtimes or environments. It is NOT used today as a direct analog of `Features`
+label, as `Feature` label indicates that the special environment configuration
+is needed to run tests in the “standard” CI.
+See the previous section and definition [here](https://github.com/kubernetes/community/blob/32a1c14d04ff78684d78b827ac7c49f70352d509/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests).
+
+The use of these labels in tests today is indicative of the labels' different
+meaning. Looking at tests configuration, wildcard `[NodeFeature:*`
+label is always used as focus and individual `NodeFeatures` are used in skip.
+It is opposite for the `Feature` label. The whildcard `[Feature:*` is always
+used to skip tests, while individual tests are present in focus.
+
+The reason for this difference is that runtimes that are being tested support
+all the NodeFeatures today and there is no need to list all NodeFeatures
+individually in test definitions. If we will have more fragmented support of
+`NodeFeatures` in runtimes being tested in CI, labels will have exactly the
+same semantics.
+
+Also both labels, `NodeFeature` and `Feature` degraded over time - they weren’t
+applied using this semantics.
+
+This KEP proposes to adjust the definition of the `Feature:` label which makes
+definitions compatible. The proposal is to unify `NodeFeatures` and `Features`
+and start relying on alternative labels for filtering in environments.
+
+- Rename NodeFeature to Feature. https://testgrid.k8s.io/sig-node-containerd#node-e2e-features will execute tests for all Features, excluding everything that needs a special environment, see labels above.
+- Document the meaning of NodeConformance. NodeConformance will indicate ALL tests that are testing enabled out of the box functionality that is not runtime or k8s distro specific. These tests may still require an additional environment set up. NodeConformance tests may have all labels specifying its environment requirements, like Slow, Disruptive, Special, etc.
```

Comment:
> These tests may still require an additional environment set up.

seems somewhat at odds with "is not runtime or k8s distro specific"

"ALL tests" seems vague, this should be all *node* tests right?

Reply:
fixed. Thank you. Original document was only covering node